arches_controlled_lists/views.py

```diff
@@ -372,3 +366,25 @@ def delete(self, request, metadata_id):
         if not count:
             return JSONErrorResponse(status=HTTPStatus.NOT_FOUND)
         return JSONResponse(status=HTTPStatus.NO_CONTENT)
+
+
+class ListOptionsView(APIBase):
+    def get(self, request):
+        node_alias = request.GET.get("node_alias")
+        graph_slug = request.GET.get("graph_slug")
+
+        controlled_list_id = (
+            NodeProxy.objects.filter(
```

Comment:
I thought we were trying to move away from proxy models?

Reply:
At least proxying our _own_ models and doing lookups that don't fallback to self, that's what I wanted to move away from. But an app proxying a model from an app it doesn't own, that I get. You can't control someone else's model.